tests/queries/models.py

```diff
@@ -328,6 +328,13 @@ def __str__(self):
         return self.name
 
 
+class SingleObjectWithoutOrdering(models.Model):
```

Comment:
I found no model that looked like it can be reused.  Maybe the test does not need to differentiate between a model with ordering in `Meta` and one via `order_by`.

@timgraham Any opinion on that?